packages/strapi-plugin-upload/admin/src/components/FiltersPicker/utils/generateNewFilters.js

```diff
@@ -0,0 +1,27 @@
+import { isArray, some } from 'lodash';
+import moment from 'moment';
+
+const generateNewFilters = (currentFilters, newFilter) => {
+  const { value } = newFilter;
+
+  let formattedValue = newFilter;
+
+  if (isArray(value)) {
```

Comment:
Can you synchronise with @HichamELBSI about this file because the filter picker needs to return a filter object instead of an array so I think the following code should be updated